deployment/v3/mosip/hsm-key-migrator/install.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Installs hsm-key-migrator
+## Usage: ./install.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+NS=hsm-key-migrator
+CHART_VERSION=0.0.1-develop
+
+echo Creating $NS namespace
+kubectl create ns $NS
+
+function installing_keymanager() {
+
+  #helm repo update
+
+  read -p "please provide module name for migration (ex: kernel, ida, esignet, etc.) : " module
+
+  if [[ -z $module ]]; then
+    echo "module is empty; EXITING;"
+    exit 0
+  fi
+
+  read -p "please provide properties file name (ex: kernel, id-authentication, esignet, etc.) : " config_prop
```

Comment:
is this the config property file name of the module for which key migration needs to be done?